apps/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -49,6 +57,11 @@ export class EnvironmentConfiguration {
     return EnvironmentConfiguration._rushTempFolderOverride;
   }
 
+  public static get absoluteSymlinks(): boolean {
```

Comment:
>absoluteSymlinks [](start = 20, length = 16)

Needs docs.  It can be `See {@link EnvironmentVariableNames.RUSH_ABSOLUTE_SYMLINKS}`